src/databricksApi/dbfs/DatabricksFSTreeItem.ts

```diff
@@ -100,12 +111,16 @@ export class DatabricksFSTreeItem extends vscode.TreeItem implements iDatabricks
 		{
 			if (action == "PREVIEW")
 			{
-				let tempFile = await Helper.openTempFile('', this.path.split('/').slice(-1)[0], false);
-				await DatabricksApiService.downloadDBFSFile(this.path, tempFile, true);
+				let localFilePath = fspath.join(
```

Comment:
if we rework the DBFS download I would also get rid of PREVIEW and SAVE and only have one code block here which downloads the file to our local sync DBFS folder and then opens it
We also would not need the saveDialog then
what do you think?

Reply:
Agree on removal of saveDialog.

However there are files that should not be opened/previewed in editor. 

It is easy for clicking on download icon as it would just download the file.

But how should Open (clicking on file name) should behave?  Should it open local copy?  Should it download and open?  I personally prefer the first option as safer one. But this is departure from current approach so I do not know.

Reply:
I would say it should behave the same way as the Workspace does:
- click/doubleclick opens the local copy
- dedicated buton to download it to the local sync DBFS folder
- maybe another dedicated button to open the save dialog and store it somewhere else

Reply:
* Done
* Done
* Save/Save As button not implemented